karma.conf.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function(config) {
+  config.set({
+      frameworks: ["jasmine", "karma-typescript"],
+      files: [
+          "src/**/*.ts" 
+      ],
+      preprocessors: {
+          "**/*.ts": "karma-typescript" 
+      },
+      reporters: ["progress", "karma-typescript"],
```

Comment:
Can you include Istanbul coverage?

Reply:
You have coverage report generated in coverage folder.